src/pe_reports/stakeholder/views.py

```diff
@@ -0,0 +1,595 @@
+            )
+        allsubs.append(subisolated)
+
+    return subdomains, allsubs
+
+
+def theaddress(domain):
+    """Get actual IP address of domain."""
+    gettheAddress = ""
+    try:
+        gettheAddress = socket.gethostbyname(domain)
+    except socket.gaierror:
+        logging.info("There is a problem with the Domain that you selected")
+
+    return gettheAddress
+
+
+def getallsubdomainIPS(domain):
+    """Get a list if ip addresses associated with a subdomain."""
+    logging.info("The domain at getallsubdomsinIPS is %s", domain)
+    alladdresses = []
+    for x in getSubdomain(domain)[0]:
+        domainaddress = theaddress(x)
+        if domainaddress not in alladdresses and domainaddress != "":
+            alladdresses.append(domainaddress)
+    return alladdresses
+
+
+def verifyIPv4(custIP):
+    """Verify if parameter is a valid ipv4 ip address."""
+    try:
+        if ip_address(custIP):
+            return True
+
+        else:
+            return False
+
+    except ValueError as err:
+        logging.error("The address is incorrect, %s", err)
+        return False
+
+
+def verifyCIDR(custIP):
+    """Verify if parameter is a valid CIDR block ip address."""
+    try:
+        if ip_network(custIP):
+            return True
+
+        else:
+            return False
+
+    except ValueError as err:
+        logging.error("The cidr is incorrect, %s", err)
+        return False
+
+
+def validateIP(custIP):
+    """
+    Verify ipv4 and cidr.
+
+    Collect address information into a list that is ready for DB insertion.
+    """
+    verifiedIP = []
+    for the_ip in custIP:
+        if verifyCIDR(the_ip) or verifyIPv4(the_ip):
+            verifiedIP.append(the_ip)
+    return verifiedIP
+
+
+def getOrganizations():
+    """Get all orgaization details from Cybersix Gill via API."""
+    url = "https://api.cybersixgill.com/multi-tenant/organization"
+
+    headers = {
+        "Content-Type": "application/json",
+        "Cache-Control": "no-cache",
+        "Authorization": f"Bearer {getToken()}",
+    }
+
+    response = requests.get(url, headers=headers).json()
+    return response
+
+
+def setNewCSGOrg(newOrgName, orgAliases, orgdomainNames, orgIP, orgExecs):
+    """Set a new stakeholder name at CSG."""
+    newOrganization = json.dumps(
+        {
+            "name": f"{newOrgName}",
+            "organization_commercial_category": "customer",
+            "countries": ["worldwide"],
+            "industries": ["Government"],
+        }
+    )
+    url = "https://api.cybersixgill.com/multi-tenant/organization"
+
+    headers = {
+        "Content-Type": "application/json",
+        "Cache-Control": "no-cache",
+        "Authorization": f"Bearer {getToken()}",
+    }
+
+    response = requests.post(url, headers=headers, data=newOrganization).json()
+
+    newOrgID = response["id"]
+
+    if newOrgID:
+        logging.info("Got here there is a new new org %s", newOrgID)
```

Comment:
This is a confusing logging message :)

Reply:
@dav3r , 
This request for change has been addresed in commit [c916f2f]([c916f2f](https://github.com/cisagov/pe-reports/pull/115/commits/c916f2f57442dec1b1af0715f3472dd5c8f871df))

Reply:
Much better, thanks!